.github/workflows/update.yml

```
@@ -8,7 +8,28 @@ on:
+      - '.github/workflows/ubuntu.yml'
+      - 'src/**'
+      - 'include/**'
+      - 'sql/**'
+      - 'pgtap/**'
+      - 'tools/testers/**'
+      - 'CMakeLists.txt'
+
+    branches-ignore:
+      - 'gh-pages'
+
+    tags: []
```

actionlint (1.7.7)

"tags" section should not be empty

(syntax-check)